createst.py

```diff
@@ -155,6 +160,23 @@ def check_requires():
         if args[item]:
             return True
 
+def check_set_args():
+    """
+    Check if the user wants midstream set to true and/or to have strict
+    checksums
+    """
+    """
+    Element order matters here. 'strictcsums' must be the last element.
+    """
+    user_args = ["midstream", "strictcsums"]
+    for item in user_args:
+        if args[item]:
+            if item == "midstream":
+                return True
+            elif item == "strictcsums":
+                return False
```

Comment:
Since we only have to treat `strictcsums` as a special case, we can probably skip the check for `midstream` as we return `True` by default from this fn.